hibike/devices/YogiBear/ui.h

```diff
@@ -0,0 +1,31 @@
+#ifndef UI_H
+#define UI_H
+
+// function prototypes
+void manual_ui();
+
+//A space for constants.
+extern float pwmInput; //Value that is received from hibike and is the goal PWM
```

Comment:
All of these `extern` variables are impossible for me to follow. Could you make the code more readable?

If `extern` cannot be suitably replaced by non-globals, you can at least:
* move all of the `extern ...;` statements into a new `globals.h` header
* include the `globals.h` header from all locations that previously defined these externs
* In the `globals.h` header, comment each variable with not only its function, but also the location where the variable is actually defined in a non-extern statement.